data/geany.glade

```diff
@@ -3427,6 +3427,23 @@
                                   </packing>
                                 </child>
                                 <child>
+                                  <object class="GtkCheckButton" id="check_always_auto_close_brackets">
+                                    <property name="label" translatable="yes">Auto-close brackets if matching bracket found</property>
```

Comment:
I tried a while to think of a line that wouldn't be too long. I can change it though.